packages/commonwealth/client/scripts/views/pages/Community/common/constants.ts

```diff
@@ -31,23 +31,24 @@ export const requirementTypes = [
   { value: TOKENS.EVM_TOKEN, label: 'EVM base tokens' },
 ];
 
-// Get eth chain id from the app.config.chains for these chains
-export const cosmosBaseChainIds = ['cosmos', 'injective'];
-export const ethBaseChainIds = ['axie-infinity', 'ethereum', 'polygon'];
-const chainIdsToFind = [...cosmosBaseChainIds, ...ethBaseChainIds];
-const foundChains = app.config.chains
-  .getAll()
-  .filter((x) => chainIdsToFind.includes(x.id.toLowerCase()));
-export const chainTypes = chainIdsToFind.map((x) => ({
-  chainBase: cosmosBaseChainIds.includes(x) ? 'cosmos' : 'ethereum',
-  value: cosmosBaseChainIds.includes(x)
-    ? 'cosmos' // cosmos and injective have 'cosmos' id
-    : foundChains.find((y) => y.id.toLowerCase() === x)?.ChainNode?.ethChainId,
-  label: x.replace(/\b\w/g, (l) => l.toUpperCase()),
-}));
-
 export const conditionTypes = [
   { value: AMOUNT_CONDITIONS.MORE, label: 'More than' },
   { value: AMOUNT_CONDITIONS.EQUAL, label: 'Equal to' },
   { value: AMOUNT_CONDITIONS.LESS, label: 'Less than' },
 ];
+
+// Get chain id's from the app.config.chains for all eth and cosmos chains
+const ethAndCosmosChains = app.config.chains
+  .getAll()
+  .filter((x) =>
+    x.base === 'ethereum'
+      ? x?.ChainNode?.ethChainId
+      : x.base === 'cosmos'
+      ? true
+      : false,
+  );
```

Comment:
In general, it would be nice to use full variable names since it's harder to scan the code with one-letter variable names